### ISSUE TITLE
Hermes UI Resign

### ISSUE THREAD

**Issue by Aahung** [HermesApp/Hermes#228]
What do you think the UI change to something similar to this (based on iTunes mini player), a little simpler, or borrow something from Pandora app on iPhone. I think: 

- the current one has too many buttons on the top with same level, 
- and the link (the three arrows) doesn't convey the right information, it they are links they may need to looks like links. 
- And, the album image's size varies, which make sometimes the progress bar too far away and too wide.
- In most cases, people use the system wide volume control instead of app one, maybe this slider should hide like iTunes.

![hermes_design based on iTunes mini player](https://cloud.githubusercontent.com/assets/3804518/7439921/4049e046-f0c8-11e4-8bc7-3e17b6dee52e.png)

**Comment by winny-** [comment]
Earlier today, I was actually thinking about some baby steps in this direction:

- Remove ZZZ (Tired of song) from default toolbar
- Remove the arrow icons and replace with an circular icon with an "i" for info, and only show the icon when hovered over.

I really your suggestions. The UI feels a complex, but has worked well thus far, and now it's time for a change to catch up with Mac OS X's aesthetic changes.

**Comment by Aahung** [comment]
Yep, the *ZZZ* button is not presented in current Pandora Web app.
Why not just place the buttons as this, users already familiar with it.
![screen shot 2015-05-02 at 1 35 00 pm](https://cloud.githubusercontent.com/assets/3804518/7440056/31505ff4-f0d0-11e4-9c29-bbca33a53d46.png)
And whole like this:
![hermes_design](https://cloud.githubusercontent.com/assets/3804518/7440075/4f193ed8-f0d1-11e4-900a-e83672df02e8.png)

**Comment by Aahung** [comment]
I think stations and histories panel can be placed like this:

1. both using same panel and use a tab switch to navigate between them.
2. (idea from Pandora iPhone App) scroll horizontally on album picture to see history, the station panel stays below.

![hermes_mocking](https://cloud.githubusercontent.com/assets/3804518/7498497/caad43c0-f453-11e4-90bd-edb43deccfeb.gif)

**Comment by nriley** [comment]
I did make the arrows only appear on mouse-over, which looks quite a lot less cluttered.  The full UI redesign will have to wait for someone who has an order of magnitude more free time than I do, I think.

**Comment by nriley** [comment]
Did a bit more cleanup in 1.2.9 — made the progress bar not appear as if you can drag it; removed some lines and tightened up the spacing a bit.  Something I'd love to do (not sure how hard it'd be; placing stuff into the title bar has recently become better supported) would be to turn the window title into a station popup, that way you didn't have to show the drawer to switch stations.
